cinema/tests/test_models.py

```diff
@@ -0,0 +1,410 @@
+        assert isinstance(country, Country)
+
+    @pytest.mark.parametrize(
+        "field_name, expected_label",
+        label_argvalues,
+        ids=label_ids
+    )
+    def test_label_of_model_field(
+            self, country, field_name, expected_label
+    ):
+        actual_label = country._meta.get_field(field_name).verbose_name
+        assert actual_label == expected_label
+
+    @pytest.mark.parametrize(
+        "field_name, option_name, expected_option_value",
+        field_argvalues,
+        ids=field_ids
+    )
+    def test_options_of_model_fields(
+            self, country, field_name, option_name, expected_option_value
+    ):
+        if option_name == "max_length":
+            actual_option_value = country._meta.get_field(field_name).max_length
+        elif option_name == "unique":
+            actual_option_value = country._meta.get_field(field_name).unique
+        else:
+            raise AttributeError(
+                f"String '{option_name}' cannot be an attribute."
+            )
+        assert actual_option_value == expected_option_value
+
+    def test_record_string_representation_is_country_name(self, country):
+        expected_result_of_method_str = country.name
+        assert str(country) == expected_result_of_method_str
+
+    def test_values_of_record_fields(self, country_factory):
+        country = country_factory(name=country_names[0])
+        actual_name = country.name
+        expected_name = country_names[0]
+        assert actual_name, expected_name
+        assert Country.objects.count() == 1
+
+
+@pytest.mark.django_db
+class TestGenreModel:
+    label_argvalues = [("name", "name"), ]
+    label_ids = [
+        f"'{arg[0]}' field: is label '{arg[1]}'?"
+        for arg in label_argvalues
+    ]
+    field_argvalues = [
+        ("name", "max_length", 16),
+        ("name", "unique", True),
+    ]
+    field_ids = [
+        f"'{arg[0]}' field: is {arg[1]} {arg[2]}?"
+        for arg in field_argvalues
+    ]
+
+    def test_record_is_instance_of_model(self, genre):
+        assert isinstance(genre, Genre)
+
+    @pytest.mark.parametrize(
+        "field_name, expected_label",
+        label_argvalues,
+        ids=label_ids
+    )
+    def test_label_of_model_field(
+            self, genre, field_name, expected_label
+    ):
+        actual_label = genre._meta.get_field(field_name).verbose_name
+        assert actual_label == expected_label
+
+    @pytest.mark.parametrize(
+        "field_name, option_name, expected_option_value",
+        field_argvalues,
+        ids=field_ids
+    )
+    def test_options_of_model_fields(
+            self, genre, field_name, option_name, expected_option_value
+    ):
+        if option_name == "max_length":
+            actual_option_value = genre._meta.get_field(field_name).max_length
+        elif option_name == "unique":
+            actual_option_value = genre._meta.get_field(field_name).unique
+        else:
+            raise AttributeError(
```

Comment:
and here strange behavior. What are you going to check here? Please tell me. Fix it through.